api-report/runtime-definitions.api.md

```diff
@@ -96,6 +96,7 @@ export type FluidDataStoreRegistryEntry = Readonly<Partial<IProvideFluidDataStor
 
 // @public
 export enum FlushMode {
+    Async = 2,
```

Comment:
I think you can have most of these changes in main, with exception of this change in enum.
I.e. you can easily have 
```ts
export enum FlushModeEx {
Async = 2,
};
```
and use it in main, and merge FlushModeEx & FlushMode in next

Reply:
How would that work? The runtime will expect FlushMode | FlushModeEx in it's APIs, right? Isn't that still a back-compat breaking change?

Reply:
No, everywhere in the code I'd keep FlushMode alone.
If caller (Loop) wants to experiment, then can do FlusModeEx.timeout as FlusModeEx.
And the only place where we need to do cast is switch:

switch (mode) {
case FlushMode.immidiate:...
case FlushModeEx.timeout as FlushMode:

}

This adds ability to push most of the code into main (which is good think from all points), while not changing officially API, but allowing (if we want to) experimentation.

Reply:
I will prepare the change for `main`

Reply:
https://github.com/microsoft/FluidFramework/pull/14106

Reply:
Ok, so the plan is to merge https://github.com/microsoft/FluidFramework/pull/14106 first, then wait for main->next integration and afterwards I'll re-adjust this PR to accommodate the changes from main. My goal is to make it so that we don't end up with merge conflicts and at the same time, next uses proper flushmode.